.drone.yml

```diff
@@ -20,7 +20,7 @@ steps:
 - name: stage-binaries
   image: rancher/dapper:v0.5.4
   commands:
-    - "Get-ChildItem -Path ./bin; Copy-Item -Force -Path ./bin/wins.exe -Destination ./package/; Get-ChildItem -Path ./package/"
+    - "Get-ChildItem -Path ./bin; Copy-Item -Force -Path ./bin/wins.exe -Destination ./package/windows/; Get-ChildItem -Path ./package/windows"
```

Comment:
Needed to move the wins.exe into ./package/windows since that's the working directory fo the Dockerfile that references it in COPY

Since this is required at every build stage, the same exact code has be duplicated across all build pipelines.